scripts/bootstrap

```diff
@@ -142,6 +142,10 @@ then
   npm install "${tgz_file}" --prefix "${static_folder}"
 fi
 
+# install cypress
+info_msg "Installing cypress"
+npm install cypress --save-dev --prefix "tests/e2e/cypress"
```

Comment:
I would not put here. I would add a new option in `./scripts/setup` to perform cypress tests and I would check if the npm install is already done or not.

Reply:
I think that this should be discussed.